testfiles/gedcom70/extensions.ged

```diff
@@ -0,0 +1,93 @@
+0 HEAD
+1 GEDC
+2 VERS 7.0
+1 SCHMA
+2 TAG _RECORD http://example.com/record-type
+2 TAG _STRUCT http://example.com/substructure-type
+2 TAG _ENUMVAL http://example.com/enumeration-value
+2 TAG _CALENDAR http://example.com/calendar
+2 TAG _MONTH http://example.com/month
+2 TAG _EPOCH http://example.com/epoch
+2 TAG _USER https://gedcom.io/terms/v7/record-SUBM
+2 TAG _CREATOR https://gedcom.io/terms/v7/SUBM
+2 TAG _CALENDRIER https://gedcom.io/terms/v7/cal-FRENCH_R
+2 TAG _JOUR https://gedcom.io/terms/v7/month-COMP
+2 TAG _CHILD https://gedcom.io/terms/v7/enum-CHIL
+2 TAG _PHRASE https://gedcom.io/terms/v7/PHRASE
+2 TAG _PARTY http://example.com/party-participation
+2 TAG _PARTY http://example.com/party
+2 TAG _BIRT https://gedcom.io/terms/v7/BIRT
```

Comment:
should be a separate URI for a record, not reuse a non-record URI

Reply:
@tychonievich Bringing the above comment back to your attention.  Do you agree or disagree?

Reply:
Discussion March 7, 2024: just delete this line and the record.